src/media/css/elements--categories.styl

```diff
@@ -63,9 +63,9 @@ mkt-category-item {
         text-decoration: none;
     }
     +media--wide-tablet() {
-        border: 1px solid rgba($greyscale-white, 0.2);
-        margin: 0;
-
+        	border: 2px solid rgba($greyscale-white, 0.2);        
+        	border-radius : 5px;
+        	margin: 0;
```

Comment:
can you remove the extra whitespace here? We keep our CSS at 4-space indents. Thanks, getting closer!